packages/ember-glimmer/tests/integration/helpers/loc-test.js

```diff
@@ -0,0 +1,33 @@
+import { RenderingTest, moduleFor } from '../../utils/test-case';
+import Ember from 'ember-metal/core';
+
+moduleFor('Helpers test: {{loc}}', class extends RenderingTest {
+
+  constructor() {
+    super();
+    this.oldString = Ember.STRINGS;
+    Ember.STRINGS = {
+      '_Howdy Friend': 'Hallo Freund'
+    };
+  }
+
+  teardown() {
+    Ember.STRINGS = this.oldString;
+  }
+
+  ['@test it lets the original value through by default']() {
```

Comment:
Seems like this only needs to check initial render and no-op rerender, not update and reset. Does that seem correct?